RGPosadas/LitterRally
# TASK-60: Authentication for Subscription

This PR closes #215.

**Steps to test**
1. Start the application.
2. Log in to MullApp.
3. Navigate to `/messages/announcements`.
4. Open your favorite GraphQL playground.
5. Send a post using the `post` mutation.
 ```graphql
    mutation Mutation($postPost: CreatePostInput!) {
      post(post: $postPost) {
        id
      }
    }
``` 
Variables:
```JSON
{
  "postPost": {
    "message": "Hello there!",
    "createdTime": "2021-02-28T16:11:00.000Z",
    "user": {
      "id": 1
    }
  }
}
```
(Don't forget the authorization header)
6. You should see the post as a JSON in the browser.
